css-four-column-layout/styles.css

```diff
@@ -0,0 +1,58 @@
+* {
+  box-sizing: border-box;
+  font-family: barlow, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  max-width: 1500px;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+}
+
+.icon-container > img {
+  width: 50px;
+  justify-content: center;
+}
+
+.screen-container {
+  text-align: center;
+}
+
+.screen-container > img {
+  text-align: center;
+  width: 100%;
+}
+
+.icon-container {
+  text-align: center;
+}
+
+.card-column h3 {
+  text-align: center;
+  margin-bottom: 2rem;
+}
+
+.card-column p {
+  text-align: center;
+}
```

Comment:
`card-column` sounds deceptively like a `column` class, which could create confusion. I would just name the class `card`.